target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -51,6 +51,18 @@ define Device/ubnt-bz
   ATH_SOC := ar7241
 endef
 
+define Device/ubnt-sw
+  $(Device/ubnt)
+  ATH_SOC := ar7242
```

Comment:
Please rename this to "SOC" due to 95030ac0b673165772daf27128359eaaf536d457.

Reply:
Done